packages/auto-approve/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "auto-approve",
+  "version": "1.0.0",
+  "description": "Approves and tags pull requests that meet user-defined criteria",
+  "private": true,
+  "author": "Google LLC.",
+  "license": "Apache-2.0",
+  "repository": "https://github.com/googleapis/repo-automation-bots.git",
+  "homepage": "https://github.com/googleapis/repo-automation-bots",
+  "bugs": "https://github.com/googleapis/repo-automation-bots/issues",
+  "main": "build/src/app.js",
+  "keywords": [
+    "probot",
+    "github",
+    "probot-app"
+  ],
+  "files": [
+    "build/src"
+  ],
+  "scripts": {
+    "compile": "tsc",
+    "start": "probot run ./build/src/auto-approve",
+    "pretest": "npm run compile",
+    "test": "cross-env LOG_LEVEL=fatal c8 mocha --exit build/test",
```

Comment:
Are you setting the LOG_LEVEL to fatal in tests so that the console output is quieter?

My only concern would be that it might sometimes make it difficult to debug. If an error is catchable, I would be tempted to use `console.warn`, rather than `console.error`, and on actual errors throw (_which you could catch in tests._).

With this approach you could avoid setting `LOG_LEVEL`.